repo/packages/N/nexus/1/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "cloud-ops@sonatype.com",
+  "website": "https://www.sonatype.com/",
+  "framework": false,
+  "description": "Nexus Repository Manager is the world's only repository manager with FREE support for popular formats. It supports Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories.",
```

Comment:
Also, can you review installation documentation at https://github.com/dcos/examples/tree/master/1.8/nexus. please update the installation steps there if anything has changed from the previous version.